roles/confluent.test/tasks/check_cluster_registry.yml

```diff
@@ -0,0 +1,47 @@
+---
+- name: Get Kafka Cluster Id
+  include_tasks: ../../tasks/rbac_setup.yml
+  vars:
+    copy_certs: false
+
+- name: Retrieve Cluster Registry details
+  uri:
+    url: "{{mds_http_protocol}}://{{ groups['kafka_broker'][0] }}:{{mds_port}}/security/1.0/registry/clusters"
+    method: GET
+    validate_certs: false
+    force_basic_auth: true
+    url_username: "{{mds_super_user}}"
+    url_password: "{{mds_super_user_password}}"
+    status_code: 200
+  register: cluster_name_query
+
+- set_fact:
+    cluster_name_query_json: "{{ cluster_name_query.json }}"
+
+- name: Assert Broker Cluster ID as expected
+  assert:
+    that:
+      - broker_id == "{{cluster_name_query_json | json_query('[0].clusterName')}}"
```

Comment:
Great work with this PR! Only minor thing is doesn't this filter have a python dependency on jmes-path:
https://docs.ansible.com/ansible/latest/user_guide/playbooks_filters.html#selecting-json-data-json-queries

Is there another way to pull out each element? maybe 
```
      - broker_id == "{{cluster_name_query_json[0].clusterName}}"
```

what does this return:
```
- debug: var= cluster_name_query_json
```

Reply:
I tried this and it didn't work, don't see why the dependency is an issue.

Reply:
what does this return:
```
- debug: var=cluster_name_query_json
```

Reply:
I've already done that.  The first thing I did was dump the contents of the var to get the structure of the json.  I then tried selecting individual elements but it was really painful to get it to work.  So then I tried the json query is it's much simpler and cleaner.

Unless there is a specific technical reason why json query is not valid here, I think it should stay as is.

Reply:
The reason not to add dependencies to our list is we will have to update the how to test docs, and potentially the jenkins image as well. 

I'd like to see the output of this:
```
- debug: var=cluster_name_query_json
```
so i can try and help!

Reply:
this works: `cluster_name_query_json[0]['clusterName']`